### PR TITLE
Motion Amendments September 2020

### PR DESCRIPTION
**Pull request by Epride**
Set of changes to the WCA Motions for Staff review. A summary of changes will be sent to Staff shortly

### CODE REVIEW

File: documents/motions/07.2020.1 - Registered Speedcubers.md
Comment (Jambrose777) [inline documents/motions/07.2020.1 - Registered Speedcubers.md:27]: What does resignation from the WCA as a registered speedcuber mean? Is this the same as the anonymization process outlined in the next point (if so, should this be contact WDPC instead of the Secretary)?

Comment (Epride) [reply documents/motions/07.2020.1 - Registered Speedcubers.md:27]: This is a change of the Motions to match the Bylaws (Specifically 2.4 resignation and termination) As we don't specifically use memberships or membership categories at the moment this doesn't really change how anything is done, just makes our documents match

File: documents/motions/16.2020.1 - Disputes.md
Comment (mirandapedro) [inline documents/motions/16.2020.1 - Disputes.md:14]: Any particular reason why this was changed? Are there situations in which these disputes shouldn't be handled by the party described in each section?

Comment (Epride) [reply documents/motions/16.2020.1 - Disputes.md:14]: Yes there are some cases where a conflict may exist for example if someone were to bring a dispute about how the Board handled the motions/Bylaws then it would be impractical for the Board to handle this case. Likewise generally if a member of the committee is involved in the dispute then it shouldn't be handled by that committee and it usually isn't. This change is mainly just to make the motions match what actually happens in practice

Comment (mirandapedro) [reply documents/motions/16.2020.1 - Disputes.md:14]: Thanks!

**Review by bruchemrfe — COMMENTED** [review]
I do not like that this is going to be removed:
   2. Acknowledged WCA Competition results and Registered Speedcuber profiles should not be removed.

The next point does not cover this. I think we should not remove results. It is like changing the past and has bad effects (for example no winner of a competition),

**Review by bruchemrfe — COMMENTED** [review]
I understand the change but I am not sure this is right, because it has the effect that board is influencing directly who has voting rights.

**Review by bruchemrfe — COMMENTED** [review]
I am not sure this is correct. It feels like being forced to dig your own grave. For staff it is different because they are enployees and may choose to leave the organisation.

**Review by bruchemrfe — COMMENTED** [review]
Isn't the Bylaws a separate document? I think there is a Bylaws document and there are Motions.

**Comment by bruchemrfe** [comment]
> @bruchemrfe I can't tell which parts some of your comments are directed toward. Can you either quote the relevant changes or add your comments directly within the files so that we can properly address them?
That is weird. I thought I actually did comment to the specific files. But I will check it again.

File: documents/motions/07.2020.1 - Registered Speedcubers.md
Comment (bruchemrfe) [reply documents/motions/07.2020.1 - Registered Speedcubers.md:27]: I do not like that this is going to be removed:
2. Acknowledged WCA Competition results and Registered Speedcuber profiles should not be removed.

The next point does not cover this. I think we should not remove results. It is like changing the past and has bad effects (for example no winner of a competition),

**Review by bruchemrfe — COMMENTED** [review]
I understand the change but I am not sure this is right, because it has the effect that Board is influencing directly who has voting rights.

File: documents/motions/09.2020.1 - Senior Delegates.md
Comment (bruchemrfe) [inline documents/motions/09.2020.1 - Senior Delegates.md:18]: I understand the change but I am not sure this is right, because it has the effect that board is influencing directly who has voting rights.

File: documents/motions/10.2020.7 - Ethics Committee.md
Comment (bruchemrfe) [inline documents/motions/10.2020.7 - Ethics Committee.md:31]: I am not sure this is correct. It feels like being forced to dig your own grave. For staff it is different because they are enployees and may choose to leave the organisation.

File: documents/motions/09.2020.1 - Senior Delegates.md
Comment (cubewhiz) [reply documents/motions/09.2020.1 - Senior Delegates.md:18]: This is just a clarification of the existing process. From Motion 08.2020.1:
6.3 Senior Delegates decide at any time which persons are proposed as WCA Delegates, then
send in an application for these persons to the WCA Board of Directors.
6.3.1 The WCA Board of Directors rejects or approves the application.
6.3.2 After approval, the Senior Delegate appoints the persons as Junior Delegates.

9.2.2 To promote a Junior Delegate to Full Delegate, the Senior Delegate must submit the
corresponding application to the WCA Board.
9.2.3 The WCA Board rejects or approves the application.
9.2.4 After approval, the Senior Delegate appoints the person as Full Delegate.

There is a similar description in other documents for promotion of Trainee and Regional Delegates, all requiring Board approval.

File: documents/motions/10.2020.7 - Ethics Committee.md
Comment (c-goodyear) [reply documents/motions/10.2020.7 - Ethics Committee.md:31]: Since WEC only deal with Staff incidents, this shouldn't be the case that Registered Speedcubers are "digging their own grave" because WEC wouldn't be investigating them. One thing to note, is that if WEC becomes non-functioning, then WDC would take over their role. If the WDC became non-functioning it would make sense to have WEC perform their role, therefore this clause would make sense.

@Epride maybe the WDC / WEC motions should be updated to show who is taking on WDC responsibilities in case of WDC being non-functioning. I would assume it's WEC, but I can see the case being made for Board as well.

Comment (cubewhiz) [reply documents/motions/10.2020.7 - Ethics Committee.md:31]: This serves two purposes:
1. Usually WEC investigates Staff conduct, so in most cases a Registered Speedcuber would be a witness, not the person being investigated.
2. In cases in which the WEC is investigating a WDC matter (due to a conflict of interest, for example), this is already a requirement for WDC investigations:

> 3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary
> Committee.

This just eliminates the discrepancy of having the process handled in two different ways depending on who conducts the investigation.

File: documents/motions/09.2020.1 - Senior Delegates.md
Comment (cubewhiz) [reply documents/motions/09.2020.1 - Senior Delegates.md:18]: > Isn't the Bylaws a separate document? I think there is a Bylaws document and there are Motions.
> https://www.worldcubeassociation.org/documents/WCA%20Bylaws.pdf
> 
> So it may be legally so that Bylaws are different from Motions.

The Bylaws themselves already describe how they should be amended:

> Except as otherwise provided by law, these Bylaws may be amended or repealed and new bylaws
adopted by approval of the Board, or by a supermajority (⅔) vote of the membership at a properly
announced meeting.

Currently, the Bylaws say they can be amended one way, but the Motions say the Bylaws are amended by a different process. We are keeping the amendment process from the Bylaws only so that the Bylaws and Motions do not contradict each other.

File: documents/motions/08.2020.2 - Trainee Delegates.md
Comment (EdHollingdale) [inline documents/motions/08.2020.2 - Trainee Delegates.md:35]:
```suggestion
      5. contribute to the timely submission of the results for the competition to the WCA Results Team
```
Remove extra spaces

**Review by c-goodyear — APPROVED** [review]
Vote passed.

**Pull request merged**
